test/assets/comparison.jl

```diff
@@ -0,0 +1,151 @@
+# Here, specialized methods for comparing models & solutions are provided.
+# The idea is not to put a big burden into the Base operators (==, isapprox)
```

Comment:
Are these implementations more efficient? What's the motivation of this?

Reply:
It has nothing to do with efficiency. When comparing models and solutions for testing, the standard operators `==` and `isapprox` aren't a good fit. They're not very configurable too. This led to the creation of new methods. Since those are just for test purposes, they're defined for the test suite only.